### ISSUE TITLE
Default deny GPIO access

### ISSUE THREAD

**Issue by marnovandermaas** [lowRISC/sonata-software#41]
Is there a way to default deny the GPIO access through the rego unless you explicitly define that the compartment has access to it?

**Comment by HU90m** [comment]
Yes this is possible with cheriot-audit's [built-in compartment package](https://github.com/CHERIoT-Platform/cheriot-audit?tab=readme-ov-file#the-compartment-package), specifically the `mmio_allow_list` rule with an empty allow set.
